pyop2/fusion/extended.py

```diff
@@ -113,7 +113,7 @@ def c_kernel_arg(self, count, i=0, j=0, shape=(0,), layers=1):
         elif self.gather == 'onlymap':
             c_args = "%s, %s" % (self.c_arg_name(i), self.c_vec_name())
         else:
-            c_args = super(FusionArg, self).c_kernel_arg(count, i, j, shape, layers)
+            c_args = super(FusionArg, self).c_kernel_arg(count, i, j, shape)
         if self.c_index:
             c_args += ", %s" % self.c_def_index()
         return c_args
```

Comment:
Reminder, add assertions that fusion never gets variable layer iteration, because it will surely break?